klient/k8s/resources/resources.go

```diff
@@ -179,3 +178,7 @@ func (r *Resources) Annotate(obj k8s.Object, annotation map[string]string) {
 func (r *Resources) Label(obj k8s.Object, label map[string]string) {
 	obj.SetLabels(label)
 }
+
+func (r *Resources) GetScheme() *runtime.Scheme {
```

Comment:
Just exposing it via a method since it had no other exported fields. This felt more consistent.